packages/flame_forge2d/test/forge2d_game_test.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flame_forge2d/flame_forge2d.dart';
+import 'package:test/test.dart';
+
+class _TestForge2dGame extends Forge2DGame {
+  _TestForge2dGame() : super(zoom: 4.0, gravity: Vector2(0, -10.0));
+}
+
+void main() {
+  group(
+    'Test corresponding position on screen and in the Forge2D world',
+    () {
+      test('Zero positioned camera should be zero in world', () {
+        expect(
+          _TestForge2dGame().screenToWorld(Vector2.zero()),
+          Vector2.zero(),
+        );
+      });
+
+      test('Zero positioned camera should be zero in FlameWorld', () {
```

Comment:
It would be good with some tests where the camera isn't positioned at the zero vector